developer_manual/basics/front-end/l10n.rst

```diff
@@ -147,11 +139,16 @@ There are global functions **t()** and **n()** available for translating strings
     n('myapp', 'Import %n calendar into {collection}', 'Import %n calendars into {collection}', selectionLength, {collection: 'Nextcloud'});
 
 
+Guidelines
+----------
+
+Please also look through the following hints to improve your strings and make them better translatable by the community
+and therefore improving the experience for non-english users.
 
-Important notes
----------------
+.. csv-table::
+   :header: "Bad","Good","Description"
 
-Please also look through the following steps to improve your strings and make them better translatable by others
+   "Don't","Do not","Using the spelled out version is easier to understand and makes translating easier"
```

Comment:
... vs …
´ vs '
user vs account
user vs people
admin vs administration
administrator vs administration
leading/trailing space
include : into translated string for french and others that prepend a space

you have more ideas @rakekniven ?